src/h5web/providers/mock/utils.ts

```diff
@@ -33,15 +38,18 @@ export function findMockEntity(root: Metadata, path: string): Entity {
   return entity;
 }
 
-export function getMockDataArray(path: string): ndarray {
-  assertAbsolutePath(path);
-
+export function findMockDataset(path: string): MockDataset {
```

Comment:
I wonder if the introduction of this function is really worth it. To me, it doesn't add much value.

Reply:
You're right 👍

I've also added `isMockDataset` with a check on `value`.